README.md

```diff
@@ -194,6 +204,53 @@ $view = $this->factory->create(EnumType::class, null, array(
 ))->createView();
 ```
 
+#### Twig extension
+
+This package come with a `enum_label` filter, available on the `EnumExtension` Twig class.
+You have to require the `twig/twig` package to get it working.
+
+The filter will try to return the constant label corresponding to the given value.
+
+It will try to translate it if possible. To enable translation, require the `symfony/translation` component
+and pass a `Symfony\Component\Translation\TranslationInterface` instance on the `EnumExtension` constructor.
+
+If not translation available, you will have the default label with class prefixing.
+
+Usage:
+
+```twig
+{{ value|enum_label('Your\\Enum\\Class') }}
+{{ value|enum_label('Your\\Enum\\Class', 'another_domain') }} {# Change the translation domain #}
+{{ value|enum_label('Your\\Enum\\Class', false) }} {# Disable translation. In this case the class prefix wont be added #}
+{{ value|enum_label('Your\\Enum\\Class', false, true) }} {# Disable translation but keep class prefix #}
+{{ value|enum_label('Your\\Enum\\Class', false, '.') }} {# Disable translation but keep class prefix with a custom separator #}
+```
+
+##### Twig extension as a service
+
+On Symfony projects, the extension can be accoladed.
```

Comment:
I meant "autoloaded".

I have no explanation except maybe just my hand failing on the keyboard like a shit.